api_spec.yaml

```diff
@@ -3246,9 +3246,204 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
```

Comment:
change config object to contain the thirdPartyId and the tenantId and remove the "config" scoping of it

Reply:
Done